lib/registries/addon/overview/route.ts

```diff
@@ -76,20 +79,39 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-            const headTags = [...this.metaTags.getHeadTags(metaTagsData)];
-            if (provider && provider.assets && provider.assets.favicon) {
-                headTags.push({
-                    type: 'link',
-                    attrs: {
-                        rel: 'icon',
-                        href: provider.assets.favicon,
-                    },
-                });
+
+            const jsonLD: any = await this.scriptTags.returnStructuredData(id);
+
+            if (jsonLD) {
+                this.set('structuredData', jsonLD);
             }
-            this.set('headTags', headTags);
+
+            const jsonString: string = this.structuredData ?
+                JSON.stringify(this.structuredData) : JSON.stringify({ isAccessibleForFree : true });
+
+            const scriptTagData = {
+                type: 'application/ld+json',
+                content: jsonString,
+            };
+
+            const metaTags: HeadTagDef[] = this.metaTags.getHeadTags(metaTagsData);
+            const scriptTag: HeadTagDef[] = this.scriptTags.getHeadTags(scriptTagData);
+            const allTags: HeadTagDef[] = metaTags.concat(scriptTag);
+
+            if (!this.currentUser.viewOnlyToken) {
+                if (provider && provider.assets && provider.assets.favicon) {
+                    allTags.push({
+                        type: 'link',
+                        attrs: {
+                            rel: 'icon',
+                            href: provider.assets.favicon,
+                        },
+                    });
+                }
+            }
+            this.set('headTags', allTags);
```

Comment:
This would not allow the code to scale well, but yes this could work for a limited application. Since we are Emberizing the pages, the script-tags service would allow us to implement it on any of the updated pages.